src/cards/ha-weather-card.html

```diff
@@ -14,23 +14,29 @@
         }
 
         .attribution {
-          color:grey;
+          color: var(--secondary-text-color);
+          text-align: right;
```

Comment:
Can we put this class in a common file? I checked out `src/resources/ha-style.html`, but I'm not sure if that's appropriate.